examples/llm/requirements.txt

```diff
@@ -11,3 +11,6 @@ omegaconf==2.2.3
 wandb==0.13.6
 pytest>=7.2.1,<8
 torchmetrics==0.11.3
+xentropy-cuda-lib@git+https://github.com/HazyResearch/flash-attention.git@v0.2.8#subdirectory=csrc/xentropy
+build==0.10.0
+ninja==1.11.1
```

Comment:
r these needed for the xentropy install? or were they needed for the dropoutresiduallayernorm stuff?